about.html

```diff
@@ -0,0 +1,138 @@
+            <li><img src="./icons/twitter_icon.png" alt="twt" ></li>
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </div>
+    <div class="second-toolbar">
+        <div class="logo"><a href="./about.html"><img src="./images/logo-removebg-preview.png" alt="main-logo" ></a></div>
+        <ul>
+            <li><a href="./index.html">Homepage</a></li>
+            <li>Program</li>
+            <li>Join</li>
+            <li><a href="#partners">Sponsor</a></li>
+            <li>News</li>
+        </ul>
+        <div class="other-element">WT Campaign</div>
+    </div>
+    <section class="homepage">
+        <nav class="navbar">
+            <button class="navbar-toggler">
+                <span class="navbar-toggler-icon"></span>
+                <span class="navbar-toggler-icon"></span>
+                <span class="navbar-toggler-icon"></span>
+            </button>
+        </nav>
+        <div class="homepage-content">
+            <h1 class="main-title">WINTRADE GLOBAL SUMMIT 2023</h1>
+            <!-- <h6>organized by: BeattheMarket</h6> -->
+            <div class="box">
+                <p>Wintrade Global Summit, held every three years, is a capital market event that embodies the slogan "Power of Trade."  </p>
+                <p>With a focus on fostering collaboration and innovation, the summit facilitates meaningful discussions that shape the future of the global economy.</p>
+                
+            </div>
+            <p>Please contact us via email if you need further information about WINTRADE GLOBAL SUMMIT</p>
+            <p class="contact">wtsummit2023@beatthemarket.com</p>
+        </div>
+    </section>
+
+    <section class="logo-section">
+        <h1>Wintrade Global Summit 2023 Logo</h1>
+        <p class="desktop">Out of numerous submissions, the chosen logo from the competition stands out as a true representation of our brand identity. Its creative design, meaningful symbolism, and harmonious colors make it instantly recognizable and memorable. </p>
+        <p class="mobile">The logo was decided through a logo competition held from Jun 1st to 10th.</p>
+        <div class="box">
+            <img src="./images/logo-removebg-preview.png">
+        </div>
+    </section>
+
+    <section class="history-section">
+        <h1>See the past WT Global Summits</h1>
+        <p>Take a look the summits which were held on London, Sydney and Baku</p>
+        <div class="card-container">
+            <div class="card">
+                <p>2014</p>
+                <p>WT Global Summit in London</p>
+            </div>
+            <div class="card">
+                <p>2017</p>
+                <p>WT Global Summit in Sydney</p>
+            </div>
+            <div class="card">
+                <p>2020</p>
+                <p>WT Global Summit in Baku</p>
+            </div>
+        </div>
+    </section>
+
```

Comment:
Kindly fix the header for this section as it is too big, add the horizontal line, and add the background color design for this and other sections of your project 👍 

Design given : 

![image](https://github.com/AliRezaBmeDu/conference-capstone/assets/20537712/2d5fccaa-9c82-4c6d-9d6c-afbbb8402bfc)

Your design :
![image](https://github.com/AliRezaBmeDu/conference-capstone/assets/20537712/7b34d474-64f2-4424-9f66-310c0b4093e3)